pkgs/applications/video/obs-studio/plugins/obs-freeze-filter.nix

```
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, obs-studio
+, qtbase
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obs-freeze-filter";
+  version = "0.3.3";
+  src = fetchFromGitHub {
+    owner = "exeldro";
+    repo = "obs-freeze-filter";
+    rev = version;
+    sha256 = "sha256-CaHBTfdk8VFjmiclG61elj35glQafgz5B4ENo+7J35o=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [
+    obs-studio
+    qtbase
+  ];
+
+  postInstall = ''
+    rm -rf "$out/share"
+    mkdir -p "$out/share/obs"
+    mv "$out/data/obs-plugins" "$out/share/obs"
+    rm -rf "$out/obs-plugins" "$out/data"
+  '';
+
+  dontWrapQtApps = true;
+
+  meta = with lib; {
+    description = "Plugin for OBS Studio to freeze a source using a filter";
+    homepage = "https://github.com/exeldro/obs-freeze-filter";
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2Only;
```

`licenses.gpl2` is deprecated.
See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206

Deciding between `gpl2Only` and `gpl2Plus` comes down whether upstream has the "or later" section.
E.g. in the header of the files, or explicitly in stated in the readme.

In this case here, it's `gpl2Only` :)

---

Done: 58582672329877bff707a44462d33cbc38285668